doc/v2/documentation_data_format.rst

```diff
@@ -309,8 +419,77 @@ Detailed field description
   ``datasetId``, which is helpful for plotting e.g. error bars.
 
 
-Observable table
-----------------
+.. _experiments_table:
+
+Experiments table
+-----------------
+
+The optional experiments table describes a sequence of different experimental
+conditions (here: discrete changes) that are applied to the model.
+
+This is specified as a tab-separated value file in the following way:
+
++---------------------+-------------------+-----------------+
+| experimentId        | time              | conditionId     |
++=====================+===================+=================+
+| PETAB_ID            | NUMERIC or '-inf' | CONDITION_ID    |
```

Comment:
NUMERIC or '-inf', see above.

Reply:
Agreed, in the measurements table it is `NUMERIC\|inf` instead